src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -9,6 +9,10 @@
       <Configuration>Debug</Configuration>
       <Platform>x64</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|Any CPU">
```

Comment:
What's the point of "Any CPU"?

Reply:
It's the default when you open VS, meaning your app targets either x64 or x86. I changed all native projects to default to x64 if Any CPU is specified.